aeron-client/src/main/c/aeron_exclusive_publication.h

```diff
@@ -52,6 +52,7 @@ typedef struct aeron_exclusive_publication_stct
     void *on_close_complete_clientd;
 
     volatile bool is_closed;
+    volatile bool revoke_on_close;
```

Comment:
Does this need to be volatile?

Reply:
I don't think that it does.  It looks like this field should only be modified by the client thread.